modules/ROOT/pages/_partials/include-install-tasks.adoc

```diff
@@ -113,3 +113,304 @@ To create the pull secret, run:
 kubectl create secret docker-registry <pull_secret> --namespace rtf --docker-server=<docker_registry_url> --docker-username=<docker_registry_username> --docker-password=<docker_ registry_password>
 ----
 // end::pullsecret[]
+
+// tag::sharedtenancy[]
+
+== (Optional) Configure Shared Tenancy
+
+If you want to configure xref:shared-tenancy.adoc[shared tenancy], add the required Kubernetes objects. The objects include:
+
+* clusterRoles/Roles
+* serviceAccounts
+* clusterRoleBindings/roleBindings
+
+. In your cluster, create a file call `configuration.yml`.
+. Copy and paste the following:
++
+[source,copy]
+----
```

Comment:
@cheungaryk I need to include this very long copy snippet, as it's meant to be copy-pasted. Is there a way to collapse it when it's rendered on the docs site, so it doesn't take so much space on the page?

Reply:
the collapse feature doesn't exist currently. If you would like to submit a feature request, please create a GUS user story for the **MS CX Engineering** product tag